vips/vip-510/bscmainnet.ts

```diff
@@ -84,14 +95,27 @@ export const Actions = {
 export const vip510 = () => {
   const meta = {
     version: "v2",
-    title: "VIP-510",
+    title: "VIP-510 [BNB Chain] [Ethereum] Deprecate low activity markets and matured PT tokens",
     description: ``,
     forDescription: "I agree that Venus Protocol should proceed with this proposal",
     againstDescription: "I do not think that Venus Protocol should proceed with this proposal",
     abstainDescription: "I am indifferent to whether Venus Protocol proceeds or not",
   };
   return makeProposal(
     [
+      // === BNB Chain ===
+      // --- Market: PT-clisBNB-APR25 on Liquid Staked BNB
+      {
+        target: COMPTROLLER_LiquidStakedBNB,
+        signature: "setCollateralFactor(address,uint256,uint256)",
+        params: [VToken_vPT_clisBNB_APR25_LiquidStakedBNB, parseUnits("0", 18), parseUnits("0.85", 18)],
+      },
+      {
+        target: COMPTROLLER_LiquidStakedBNB,
+        signature: "setActionsPaused(address[],uint8[],bool)",
+        params: [[VToken_vPT_clisBNB_APR25_LiquidStakedBNB], [Actions.MINT, Actions.ENTER_MARKET], true],
+      },
```

Comment:
Not needed. These commands were already executed in the [VIP-503](https://app.venus.io/#/governance/proposal/503?chainId=56)

Reply:
Removed [5f776aa](https://github.com/VenusProtocol/vips/pull/567/commits/5f776aa0e550a892820d4dac4f16d5d13dadbbd4)